tests/test-daemon-integration.py

```diff
@@ -33,7 +33,7 @@ class TestDaemonIntegration(dbusmock.DBusTestCase):
     _MOCK_EVENT_NOTHING_HAPPENED = '5071dd96-bdad-4ee5-9c26-3dfef34a9963'
     _MOCK_EVENT_NOTHING_HAPPENED_BYTES = \
         uuid.UUID(_MOCK_EVENT_NOTHING_HAPPENED).bytes
-    _NANOSECONDS_PER_SECOND = 1000000000L
+    _NANOSECONDS_PER_SECOND = 1000000000
```

Comment:
optional: I always prefer using `1e9` directly in code rather than defining constants for SI prefixes.